### DISCUSSION TITLE
Strange that I cannot choose from any Wine version - so that it will be fetched automatically :(

### DISCUSSION THREAD

**Discussion by vitaly-zdanevich** [lutris/lutris#5635] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/e8e19807-3a78-4ca1-b98b-cae3e70ebdd8)

My game is broken in one aspect (camera rotation in Night Watch), so I want to try a few Wine versions - so do I need manually download them from Wine GitHub and put to some folder? Why no automatization for that?

**Comment by danieljohnson2** [comment]
No, you can download Wine versions with Lutris. There's an icon next to Wine in the sidebar that brings up this:

![image](https://github.com/user-attachments/assets/fd1f5b92-0339-4142-9428-5dea7cfecc4f)

You can install Wine versions there.

**Reply by vitaly-zdanevich** [reply]
Thanks, but here I see only one version
![image](https://github.com/user-attachments/assets/6fb47e48-570e-475e-88df-8f2d07f58663)

**Reply by danieljohnson2** [reply]
I see 10 in that screenshot. I believe you can scroll down for more.

Click one of those install buttons. Embrace the magic!

**Reply by vitaly-zdanevich** [reply]
On that window - only one wine and many lutris... Lutris also mean wine?

**Reply by danieljohnson2** [reply, marked as answer]
These are different Wine builds. Some were built by the Lutris project and are so named. More recently we've been using @GloriousEggroll's builds, and that's what ``wine-ge-8-26`` is. That is the recommended one.

But there are lots of old versions and weird specialized builds in there.
